docs/Release_Guide/release_steps/met/create_release_reference_branch.rst

```diff
@@ -9,7 +9,7 @@ For rc1 development releases, create a new reference branch for the upcoming off
     new official release and push it to GitHub. The branch name should match
     the format 'main_vX.Y-ref' where X.Y is the major/minor release number.
 
-  .. parsed-literal::
+    ::
```

Comment:
`parsed-literal` needs to be here. Otherwise the `|projectRepo|` substitution does not occur properly. See https://metplus--3124.org.readthedocs.build/en/3124/Release_Guide/met_development.html#create-release-branch